awx/ui/client/src/inventories-hosts/inventories/related/sources/sources.form.js

```diff
@@ -340,6 +300,36 @@ export default ['NotificationsList', 'i18n', function(NotificationsList, i18n){
                 ngDisabled: '!(inventory_source_obj.summary_fields.user_capabilities.edit || canAdd)',
                 subForm: 'sourceSubForm'
             },
+            host_filter: {
+                label: i18n._("Host Filter"),
+                type: 'text',
+                dataTitle: i18n._('Host Filter'),
+                dataPlacement: 'right',
+                awPopOver: "<p>" + i18n._("Regular expression where only matching hosts will be imported.") + "</p>",
```

Comment:
I think it would be good to mention that this is client-side filtering.

I'm still very worried about confusion with these fields, because there is duplication between them and the inventory plugin options themselves. I wouldn't even hesitate saying that "this is not an inventory plugin parameter"

Reply:
updated:
![Screenshot from 2020-08-27 12-18-48](https://user-images.githubusercontent.com/9753817/91468222-ad954700-e85f-11ea-931d-3bcb68c76f08.png)
![Screenshot from 2020-08-27 12-19-01](https://user-images.githubusercontent.com/9753817/91468232-b0903780-e85f-11ea-9867-48d3daac2783.png)
![Screenshot from 2020-08-27 12-19-13](https://user-images.githubusercontent.com/9753817/91468241-b38b2800-e85f-11ea-976c-75bba2b232cf.png)

cc @dsesami